tests/MongoDB.Driver.Tests/Specifications/server-discovery-and-monitoring/ServerDiscoveryAndMonitoringIntegrationTestRunner.cs

```diff
@@ -30,21 +31,25 @@
 
 namespace MongoDB.Driver.Tests.Specifications.server_discovery_and_monitoring
 {
-    public class ServerDiscoveryAndMonitoringIntegrationTestRunner : DisposableJsonDrivenTestRunner, IJsonDrivenTestRunner
+    public class ServerDiscoveryAndMonitoringIntegrationTestRunner : MongoClientJsonDrivenTestRunnerBase, IJsonDrivenTestRunner
```

Comment:
This class should probably be `IDisposable` because you got rid of `DisposableJsonDrivenTestRunner`.

Reply:
Or don't get rid of `DisposableJsonDrivenTestRunner`?

Reply:
>Or don't get rid of DisposableJsonDrivenTestRunner?

Done